js/scripts1.js

```diff
@@ -0,0 +1,58 @@
+class General {
```

Comment:
too generic name. It's better rename to something like `MenuItem`

Reply:
Vladislav, thank you for all noties, I'll correct everything